storage/innobase/handler/ha_innodb.cc

```diff
@@ -11860,18 +11919,6 @@ create_table_info_t::create_options_are_invalid()
 		return("COMPRESSION");
 	}
 
-	/* Check the encryption option. */
-	if (ret == NULL && m_create_info->encrypt_type.length > 0) {
```

Comment:
Yes. I actually added comment for it. "Validate encryption
parameter even if `strict_mode` is OFF." Invalid compression option
with `strict_mode=OFF` does not make an error, while invalid
encryption option does. It was original documented MySQL behaviour
and I kept it. Fun part is that invalid encryption in vanilla server
fails in other place, not here :)